doc/content/getting-started/migrating/migrating-from-v2.md

```diff
@@ -64,7 +84,7 @@ $ ttn-lw-migrate devices --dry-run --verbose --source ttnv2 "mydevice" > devices
 $ ttn-lw-migrate device --source ttnv2 "mydevice" > devices.json
 ```
 
-{{< warning >}} The export process will clear the device root keys (**AppKey**) and session (**AppSKey**, **NwkSKey**, **DevAddr**, **FCntUp** and **FCntDown**) from {{% ttnv2 %}}. Execute any commands with the `--dry-run` first, which will do everything except delete the device keys from {{% ttnv2 %}}. {{</ warning >}}
+{{< warning >}} The export process will clear the device root keys (**AppKey**) and session (**AppSKey**, **NwkSKey**, **DevAddr**, **FCntUp** and **FCntDown**) from {{% ttnv2 %}} by default. You can disable this by using the  `--ttnv2.with-session=false` flag. {{</ warning >}}
```

Comment:
Why not use `--dry-run` ?

Reply:
Regarding the `--dry-run` flag, the usage is already mentioned in the same section.

Reply:
Right, it makes sense to say to use that and then use it in the example. I don't think it make sense to say

"disable this by using the  `--ttnv2.with-session=false` flag"

and then

```
$ ttn-lw-migrate device --source ttnv2 --dry-run "mydevice" > devices.json
``` 

that is confusing. Let's stick to one. @neoaggelos recommended --dry-run, so i prefer that.

Reply:
@benolayinka, By default the export process will export session information of the device and clears off all the info from V2. If the `--ttnv2.with-session=false` flag is used, it will not include session info in the exported file and doesn’t delete it from V2.

So if the customer wants to export their devices without session info, `--ttnv2.with-session=false` flag needs to be added in the command. Hence, the purpose of this is different from that of the  `--dry-run` flag.